Sources/Tracking/CustomerIO.swift

```diff
@@ -230,7 +230,6 @@ public class CustomerIO: CustomerIOInstance {
         // run cleanup in background to prevent locking the UI thread
         threadUtil.runBackground { [weak self] in
             self?.cleanupRepository?.cleanup()
-            self?.cleanupRepository = nil
```

Comment:
How about updating the comment above since there is no call to the method `cleanup` now.